src/ln/channel.rs

```diff
@@ -1092,28 +1167,50 @@ impl Channel {
 	// Message handlers:
 
 	pub fn accept_channel(&mut self, msg: &msgs::AcceptChannel) -> Result<(), HandleError> {
+		macro_rules! send_error_message {
+			( $msg: expr ) => {
+				return Err(HandleError{err: $msg, action: Some(msgs::ErrorAction::SendErrorMessage{ msg: msgs::ErrorMessage { channel_id: msg.temporary_channel_id, data: $msg.to_string() }})});
```

Comment:
Technically its an existing bug, but it'd be nice to handle the Err return in ChannelManager. It should fail the channel in that case, but it currently doesn't.